README.md

```diff
@@ -45,9 +58,11 @@ rustup update
 Once done, finish installing the support software:
 
 ```bash
-sudo apt install make clang pkg-config libssl-dev
+sudo apt install make clang pkg-config libssl-dev libclang-dev
```

Comment:
looking at https://getsubstrate.io/
> $MAKE_ME_ROOT apt install -y cmake pkg-config libssl-dev git gcc build-essential git clang libclang-dev

(note that `git` is repeated twice, also `gcc` is part of `build-essential`)

I would change it to 
```suggestion
sudo apt install build-essential git cmake clang libclang-dev pkg-config libssl-dev 
```